Automattic/wp-calypso
# Reader: Remove old unused icons

#### Proposed Changes

This PR removes a few old Reader icons that became unused and obsolete since #70659.

Blocked by https://github.com/Automattic/wp-calypso/pull/71010.

#### Testing Instructions

Verify the removed icon components are not in use. Usually, it's enough to verify that there are no bundle size changes reported by the bot.
